session/bootstrap_test.go

```diff
@@ -60,10 +60,14 @@ func (s *testBootstrapSuite) TestBootstrap(c *C) {
 	c.Assert(se.Auth(&auth.UserIdentity{Username: "root", Hostname: "anyhost"}, []byte(""), []byte("")), IsTrue)
 	mustExecSQL(c, se, "USE test;")
 	// Check privilege tables.
-	mustExecSQL(c, se, "SELECT * from mysql.global_priv;")
-	mustExecSQL(c, se, "SELECT * from mysql.db;")
-	mustExecSQL(c, se, "SELECT * from mysql.tables_priv;")
-	mustExecSQL(c, se, "SELECT * from mysql.columns_priv;")
+	rs := mustExecSQL(c, se, "SELECT * from mysql.global_priv;")
+	rs.Close()
```

Comment:
Is it allowed to ignore `err`?

Reply:
`mustExecSQL` panics on error... 
oh, you mean `rs.Close` return error? The lint do not check ignore error for test code. There are a lot of places ignore the error